source/chapters/1/security.md

```diff
@@ -1,57 +1,52 @@
 ---
 title: Security
-description: Let's talk Security
+description: Let's talk about Security
 disqusPage: 'Chapter 1: Security'
 ---
 
-Security is a very important aspect of any application. When your code-base grows,
-you need to more and more careful on how you handle this. We'll show you first how we can
-secure our methods and publications, then you will be shown some tips and tricks to handle 
-it for an evergrowing code base.
+Security is a very important aspect of any application. When your codebase grows, security must be one of the core aspects 
```

Comment:
I wouldn't redirect the users to that page. We should focus more on Meteor, what they can do and how they can protect  server side